docs/en/integrations/language-clients/java/index.md

```diff
@@ -5,126 +5,250 @@ keywords: [clickhouse, java, jdbc, client, integrate, r2dbc]
+|DateTime32             |✔                    |✔                    |
+|DateTime64             |✔                    |✔                    |
+|Interval               |✗                    |✗                    |
+|Enum                   |✔                    |✔                    |
+|Enum8                  |✔                    |✔                    |
+|Enum16                 |✔                    |✔                    |
+|Array                  |✔                    |✔                    |
+|Map                    |✔                    |✔                    |
+|Nested                 |✔                    |✔                    |
+|Tuple                  |✔                    |✔                    |
+|UUID                   |✔                    |✔                    |
+|IPv4                   |✔                    |✔                    |
+|IPv6                   |✔                    |✔                    |
+|Object                 |✗                    |✔                    |
+|Point                  |✔                    |✔                    |
+|JSON                   |✔                    |✔                    |
+|Nothing                |✔                    |✔                    |
+|MultiPolygon           |✔                    |✔                    |
+|Ring                   |✔                    |✔                    |
+|Polygon                |✔                    |✔                    |
+|SimpleAggregateFunction|✔                    |✔                    |
+|AggregateFunction      |✗                    |✔                    |
 
-### Environment requirements
+:::note
+- AggregatedFunction - :warning: does not support `SELECT * FROM table ...`
+- Decimal - `SET output_format_decimal_trailing_zeros=1` in 21.9+ for consistency
+- Enum - can be treated as both string and integer
+- UInt64 - mapped to `long` in client-v1 
+:::
 
-- [OpenJDK](https://openjdk.java.net) version >= 8
+### Compatibility
+
+- All projects in this repo are tested with all [active LTS versions](https://github.com/ClickHouse/ClickHouse/pulls?q=is%3Aopen+is%3Apr+label%3Arelease) of ClickHouse.
+- [Support policy](https://github.com/ClickHouse/ClickHouse/blob/master/SECURITY.md#security-change-log-and-support)
+- We recommend to upgrade client continuously to not miss security fixes and new improvements
+  - If you have an issue with migration - create and issue and we will respond! 
 
-### Compatibility with ClickHouse
+## Client-V2
 
-| Client version | ClickHouse |
-|----------------| ---------- |
-| 0.6.0          | 22.8+      |
+Implementation of a new API. It uses Apache Http Client to communicate with ClickHouse server. We have selected this http client because it has many built-in features and 
+has proven itself in old client implementation. We are planning to support other http client libraries. 
 
-### Installation
+*Note*: Client-V2 is currently in the phase of active development and we are still working on it. 
```

Comment:
Im not sure what it means for the end user: that the v2 is not stable?

Reply:
I think we need to discuss it.

Reply:
I've remove this note